python/vowpalwabbit/pyvw.py

```diff
@@ -277,6 +277,10 @@ class vw(pylibvw.vw):
     """The pyvw.vw object is a (trivial) wrapper around the pylibvw.vw
     object; you're probably best off using this directly and ignoring
     the pylibvw.vw structure entirely."""
+    log_wrapper = None
```

Comment:
Does this work outside of the `__init__` function?

Reply:
yes, it is called class attributes

Reply:
actually I see that I haven't moved all of them here (at least forgot log_fwd). If approach is good, will send update and make it more consistent

Reply:
Which version were that added in?

Reply:
I do not see version where it is unavailable. 2.7, 3.5+ all have it.

Reply:
Awesome!